src/msg/async/AsyncMessenger.cc

```diff
@@ -171,13 +171,16 @@ void Processor::accept()
     Worker *w = worker;
     if (!msgr->get_stack()->support_local_listen_table())
       w = msgr->get_stack()->get_worker();
+    else
+      ++w->references;
     int r = listen_socket.accept(&cli_socket, opts, &addr, w);
```

Comment:
the reference doesn't mean the lifecycle of Worker, it's just a hit about connection aggresive